modules/preferences/test/spec.coffee

```diff
@@ -5,6 +5,14 @@ describe 'hx-preferences', ->
       expect(hx.preferences.supportedLocales(list)).toEqual(hx.preferences)
       expect(hx.preferences.supportedLocales()).toEqual(list)
 
+    it 'locale: should not be possible to explicitly clear the locale', ->
+      # sanity check
+      expect(hx.preferences.locale()).toBeDefined()
+
+      hx.preferences.locale undefined
+      expect(hx.preferences.locale()).toBeDefined()
```

Comment:
Should this not be set back to the default locale (e.g. `navigator.language`) rather than seemingly doing nothing?